### DISCUSSION TITLE
Bump some deps version

### DISCUSSION THREAD

**Discussion by Mister-Hope** [vuepress/core#207] [Q&A] [answered]
I would be happy if VuePress V2 can come with same deps in latest version when officially touch stable.😉 So that I won't worry it about issues with mutiple version if I am using latest deps in my own theme and plugins.

E.g.: bump to sass-loader@v12 in theme-default as it's installed by default and maybe others like postcss-loader.

I think a minium requirement of 12.13.0 is ok.

**Comment by meteorlxy** [comment, marked as answer]
Always check the codebase first

https://github.com/vuepress/vuepress-next/blob/ab5edcfe18f06500ab99637b859b56831bdefdc7/packages/%40vuepress/theme-default/package.json#L46

I'm upgrading version once finding some deps need to be updated. You can open a PR for that
